x/msgfee/antispam_query.go

```diff
@@ -0,0 +1,30 @@
+package msgfee
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/errors"
+)
+
+var _ weave.QueryHandler = (*AntiSpamQuery)(nil)
+
+// AntiSpamQuery allows querying currently set anti-spam fee
+type AntiSpamQuery struct {
+	minFee coin.Coin
+}
+
+func NewAntiSpamQuery(minFee coin.Coin) *AntiSpamQuery {
+	return &AntiSpamQuery{minFee: minFee}
+}
+
+func (q *AntiSpamQuery) Query(db weave.ReadOnlyKVStore, mod string, data []byte) ([]weave.Model, error) {
```

Comment:
Okay, so it ignors mod and data, and anything returns the same value.
Could enforce `mod == "" && len(data) == 0` but I guess this is more user-friendly.

Nice implementation by the same. Simple and isolated from other code.